fusedrug/utils/file_formats/indexed_fasta_custom.py

```
@@ -98,6 +99,16 @@ def __init__(
                     print(f"index file already found: {index_filename}")
                     already_found = True
 
+            if already_found:
+                # Fasta file should be older than it's corresponding index file
+                fasta_file_creation_time = os.path.getctime(self.filename)
+                index_file_creation_time = os.path.getctime(self.index_filename)
+
+                if fasta_file_creation_time > index_file_creation_time + 42:
+                    warnings.warn(
```

Maybe raise an error and suggest deleting the index file to recreate it?